master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/MetaHandler.java

```diff
@@ -141,16 +141,6 @@ public ResourceResponse handleWriteRequest(ResourceProtos.ResourceRequest reques
           metaSystem.updateWorkerLostMeta(host, rpcPort, pushPort, fetchPort, replicatePort);
           break;
 
-        case WorkerRemove:
```

Comment:
For back compatibility, I think we can't remove `WorkerRemove` msg, or it will throw Exception. We need to remain this message for at least one minor version. We can delete the case block.

Reply:
Done. Thanks suggestions from @waitinfuture, @RexXiong, PTAL if have time.